### DISCUSSION TITLE
language.toml documentation error

### DISCUSSION THREAD

**Discussion by HyperAlch** [helix-editor/helix#5588] [General]
I needed to setup the gdscript language server on Windows 10. I followed the guide below:
https://github.com/helix-editor/helix/wiki/How-to-install-the-default-language-servers#gdscript

It's a great tutorial, but I believe a small note telling Windows users to add the nmap directory to the Windows Path after running
`winget install nmap` would be beneficial, as it seems that winget doesn't do this automatically.  As it is written right now, the gdscript language server tutorial may not work for some Windows users. I think a note telling people to make sure the nmap directory is in their PATH would save a lot of people time as I had to figure this out on my own, which used up a good hour of my time. 

Maybe I'm just a noob when it comes to Windows and this is something everyone should know. But in linux when you install something via a package manager, that program is automatically put on the PATH after the install. This seems not to be the case with `winget` .

**Comment by archseer** [comment]
The wiki is user-editable so you could add a note to that section?

**Reply by HyperAlch** [reply]
Sure, no problem. I didn't want to over step any bounds or be disrespectful to the people who have taken the time to write the guide. But if I have permission to do so I'll edit it. Thanks for your time

**Reply by goyalyashpal** [reply]
Just for records:

A little note to help Windows users with PATH issues
https://github.com/helix-editor/helix/wiki/How-to-install-the-default-language-servers/_compare/69e91118eca90016cb3354d848d4e046e4ee3dd1...c045eb5f8415628e56a00d13685a0960bcb5a6c8
